packages/engines/core-engine/src/game-engine/engines/lorcana/src/cards/definitions/008/actions/041-only-so-much-room.ts

```diff
@@ -14,7 +14,7 @@ export const onlySoMuchRoom: LorcanaActionCardDefinition = {
   abilities: [
```

Comment:
This card has only one ability with two effects, not 2 abilitites with one effect each